src/Commands/Self/Plugin/InstallCommand.php

```diff
@@ -42,6 +43,38 @@ public function install(array $projects)
         }
     }
 
+    /**
+     * Migrate Terminus 2 plugins.
+     *
+     * @command self:plugin:migrate
+     * @aliases plugin:migrate
+     *
+     * @throws \Pantheon\Terminus\Exceptions\TerminusException
+     */
+    public function migrate()
```

Comment:
Every command should be in its own file. Please move this to a MigrateCommand.php file. If you need to call a function from InstallCommand, consider moving it to the parent class

Reply:
Fair enough.

Reply:
I advised @uberhacker to not follow this convention here. There's a number of things that would need to move to another class, and it doesn't seem right putting all of the install functionality in the base class.

If anyone feels strongly about the one-command-per-file convention, we can consider a refactor in another PR as follow-on work.

Reply:
There is plenty of clean up work that needs to be done and refactoring can certainly be part of that effort.  I noticed there were duplicate validation functions in various places.  Probably only need one in PluginBaseCommand.php.  Also, several utility functions will be deprecated in Terminus 3.